### PR TITLE
Review notes regarding Basic Workflow mit GitHub

### PR DESCRIPTION
**Pull request by TobiasSteinhagen**

### CODE REVIEW

File: note.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Basic Git Workflow mit GitHub :github:
+Once
+In the shell
+git init # create a repo (once per project)
+# don't forget to add a .gitignore file
```
Comment (Pia-Lueling) [inline note.md:5]: Don`t forget it ;)

File: note.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+git push -u origin "branchname" # push to GitHub
+# continue on GitHub
+On GitHub
+- Create a pull request for that branch (to merge it into main)
+- Send a link to the changed files to your team
+- Get a code review
+- Optional: implement changes, push again to update pull request
+- Merge and delete the branch on GitHub
+- Continue in the shell
+Again in the shell
```
Comment (bermyu) [inline note.md:23]: well done!

File: note.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+git switch main
+git pull # get the merged commits in your local main
+git branch -d "oldbranchname" # delete the merged branch
```
Comment (Pia-Lueling) [inline note.md:26]: LGTM 😇

**Comment by DavevonH** [comment]
LGTM 😎

**Comment by Pia-Lueling** [comment]
Wunderbar 😌

**Comment by bermyu** [comment]
Well done!

File: note.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+Basic Git Workflow mit GitHub :github:
+Once
+In the shell
+git init # create a repo (once per project)
+# don't forget to add a .gitignore file
+git remote add origin "ssh-path-to-github"
+For each feature
+In the shell
```
Comment (DavevonH) [inline note.md:8]: good shell

**Review by Pia-Lueling — APPROVED** [review]
LGTM

**Review by DavevonH — COMMENTED** [review]
Nice

**Comment by TobiasSteinhagen** [comment]
Thx for your feedback

**Pull request merged**